source/data/tier1data/basics.rst

```diff
@@ -0,0 +1,12 @@
+######################
+Basics
+######################
+
+.. toctree::
+   :maxdepth: 2
+
+   basics/introduction_to_tier1data
+   basics/architecture
+   basics/user_access
+
```

Comment:
I'd prefer to keep it like this because we see it as different topics (hence want to separate it) and to keep consistency with https://rdm-docs.icts.kuleuven.be/mango/index.html. But if it is really too complicated now, I can do it

Reply:
Applied